app/js/features/metadata/saga.js

```diff
@@ -0,0 +1,53 @@
+import MetadataStore from 'Embark/contracts/MetadataStore';
+import {fork, takeEvery, put, all} from 'redux-saga/effects';
+import {
+  LOAD, LOAD_SELLER, LOAD_SELLER_FAILED, LOAD_SELLER_SUCCEEDED, LOAD_OFFERS_SUCCEEDED, LOAD_OFFERS_FAILED, LOAD_OFFERS
+} from './constants';
+
+export function *loadSeller({address}) {
+  try {
+    const size = yield MetadataStore.methods.sellersSize().call();
+    if (size === '0'){
+      return;
+    }
```

Comment:
Is the size check useful? Because knowing that the size is greater than 0 doesn't mean the address specified is a seller anyway.

Reply:
Interesting: the size check is annoying as if there is no seller, the following call fails
I could let it failed but it looked bad too so I made this extra call.

What do you think?

Reply:
Ah ok, thanks for clarifying. I did think it could be something like that. Maybe the array is not initiated if there is no seller. 
Anyway, it's no big deal, just wanted to know